testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts

```diff
@@ -1,69 +1,165 @@
+    );
+  });
+
+  it("Validate user is able to be searchable by BCeID + UserId", () => {
+    manageUsersObject.inputUserIdOnAddUserModal().click();
+    cy.contains("SIMS").should("be.visible");
+  });
+
+  it("Search for the users no user is returned", () => {
+    manageUsersObject.inputUserIdOnAddUserModal().type("SIMS-ON");
+    cy.contains("No data available").should("not.be.visible");
+  });
+});
+
+describe("Manage Users - edit user modal", () => {
+  before(() => {
+    institutionHelperActions.loginIntoInstitutionSingleLocation();
     dashboardInstitutionObject.manageInstitutionButton().click();
-    manageUsersObject.manageUsersButton().click({ force: true });
-    manageUsersObject.addNewUserButton().click();
-    manageUsersObject.selectUserDropdown().click();
-    manageUsersObject.filterInputName().type("SIMS COLLF").type("{enter}");
-    manageUsersObject.addUserButton().click();
-    manageUsersObject.addUserToAccountMessage().should("be.visible");
-    manageUsersObject.cancelButton().click();
+    manageUsersObject.manageUsersButton().click();
   });
 
-  it("Verify that search bar is working properly by searching incorrect word", () => {
+  it("Validate edit user default modal", () => {
+    manageUsersObject.clickOnEdit("email");
+    manageUsersObject.editUserModal().should("be.visible");
+    manageUsersObject.isAdminRadioButton().should("be.checked");
+    manageUsersObject.isAdminRadioButton().should("be.unchecked");
+    manageUsersObject.editUserNowButton().should("be.visible");
+    manageUsersObject.cancelButton().should("be.visible");
+  });
+});
+
+describe("Manage Users - Adding a user", () => {
+  before(() => {
+    institutionHelperActions.loginIntoInstitutionSingleLocation();
     dashboardInstitutionObject.manageInstitutionButton().click();
-    manageUsersObject.manageUsersButton().click({ force: true });
-    manageUsersObject.searchUserInputText().type("Dummy data");
-    manageUsersObject.searchButton().click();
-    manageUsersObject.noRecordsFoundMessage().should("be.visible");
-    manageUsersObject.zeroUserContains().should("be.visible");
+    manageUsersObject.manageUsersButton().click();
+  });
+
+  it("Validate adding user to a location", () => {
+    manageUsersObject.addNewUserButton();
+    manageUsersObject
+      .inputUserIdOnAddUserModal()
+      .type("cypressautomation@auto.test");
+    manageUsersObject.userLocationAccess().click();
+    manageUsersObject.addUserNowButton().click();
+    manageUsersObject.getUsersList().eq(3);
+  });
+
+  it("Validate adding user to as an admin", () => {
+    manageUsersObject.addNewUserButton();
+    manageUsersObject
+      .inputUserIdOnAddUserModal()
+      .type("cypressautomation@auto.test");
+    manageUsersObject.isAdminRadioButton().click();
+    manageUsersObject.isLegalSigningAuthorityButton().click();
+    manageUsersObject.addUserNowButton().click();
+    manageUsersObject.getUsersList().eq(3);
+  });
+
+  it("Validate Edit user ", () => {
```

Comment:
There is an extra space at the end of the test description, please remove it.

Reply:
Done